FWCore/Framework/src/SubProcess.cc

```diff
@@ -434,7 +446,8 @@ namespace edm {
                                      *schedule_,
                                      rp,
                                      ts,
-                                     esp_->eventSetup(),
+                                     esp_->eventSetupImpl(),
```

Comment:
Why is the `EventSetupImpl` not gotten via the call `*iEventSetupImpls->at(esp_->subProcessIndex())` like how it was done in processOneEventAsync?

Reply:
See comment above